Adafruit_LSM6DS.cpp

```diff
@@ -52,20 +61,45 @@ Adafruit_LSM6DS::~Adafruit_LSM6DS(void) {
  *   @param sensor_id Optional unique ID for the sensor set
  *   @returns True if chip identified and initialized
  */
-bool Adafruit_LSM6DS::_init(int32_t sensor_id) { return false; };
+bool Adafruit_LSM6DS::_init(int32_t sensor_id) {
+  (void)sensor_id;
+
+  // Enable accelerometer with 104 Hz data rate, 4G
+  setAccelDataRate(LSM6DS_RATE_104_HZ);
+  setAccelRange(LSM6DS_ACCEL_RANGE_4_G);
```

Comment:
Arduino official use Accel 4G and Gypro 2000 DPS across their IMU librraries. Setting it on init() to make sure the value returned from compatible API is matched with using Arduino lib (tested on actual hardware).